pkg/synthetictests/networking.go

```diff
@@ -104,6 +93,19 @@ func testPodSandboxCreation(events monitorapi.Intervals, clientConfig *rest.Conf
 			continue
 		}
 		if strings.Contains(event.Message, "pinging container registry") && strings.Contains(event.Message, "i/o timeout") {
+			if platform == "" {
```

Comment:
`platform` will be always an empty string here.

Reply:
It is defined [here](https://github.com/openshift/origin/pull/27647/files#diff-1fd611ecbabf8351af6690c1f2a51335b15d052c7fbdf288686335f87c9528b8R96) and then grabs a value [here](https://github.com/openshift/origin/pull/27647/files#diff-1fd611ecbabf8351af6690c1f2a51335b15d052c7fbdf288686335f87c9528b8R105) for the next for loop iterations.